components/code-editor/code-editor.spec.ts

```diff
@@ -33,20 +31,20 @@ export class NzTestCodeEditorFullControlComponent {
 }
 
 describe('code editor', () => {
-  describe('basic', () => {
-    // let fixture: ComponentFixture<NzDemoCodeEditorBasicComponent>;
+  // describe('basic', () => {
```

Comment:
why skip this test? what about delete it directly?

Reply:
As you can see, this is an empty test, which is not allowed in the latest jasmine.

Reply:
I chose to annotate it so we can come back and refine it later. Of course, I could have just deleted it.